utils/zerovec/src/ule/chars.rs

```diff
@@ -108,8 +104,8 @@ mod test {
 
     #[test]
     fn test_parse() {
-        // 1-byte, 2-byte, 3-byte, and 4-byte character in UTF-8 (not as relevant in UTF-32)
-        let chars = ['w', 'ω', '文', '𑄃'];
+        // 1-byte, 2-byte, 3-byte, and two 4-byte character in UTF-8 (not as relevant in UTF-32)
```

Comment:
Nit: what's the reason for adding another 4-byte character? It makes this test slightly irregular which might be confusing to future readers (i.e. are emoji special?).

Reply:
Nothing special with emoji. It's more about the size of the array. Since I'm covering modulo 4 to modulo 3, i wanted the size to be something not divisible by either 3 or 4 so that it doesn't effect the test

Reply:
Yeah I think it's fine to have two